src/botservice/azext_bot/botservice/operations/bots_operations.py

```diff
@@ -62,7 +62,7 @@ def create(
         # Construct URL
         url = '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.BotService/botServices/{resourceName}'
         path_format_arguments = {
-            'resourceGroupName': self._serialize.url("resource_group_name", resource_group_name, 'str'),
+            'resourceGroupName': self._serialize.url("resource_group_name", resource_group_name, 'str', max_length=64, min_length=2, pattern=r'^[a-zA-Z0-9][a-zA-Z0-9_.-]*$'),
```

Comment:
Is this newly generated from autorest? or are these manual changes?

Reply:
newly generated from autorest. i added some validation to the resource group parameter in the swagger , so it showed up here